e2e/packages/server-extensibility/src/server.ts

```diff
@@ -0,0 +1,40 @@
+import { get } from "koa-route";
+import Root from "./components";
+import type ServerExtensibility from "../types";
+
+const serverExtensibility: ServerExtensibility = {
+  name: "e2e-server-extensibility",
+  roots: {
+    serverExtensibility: Root,
+  },
+  server: {
+    serverExtensibility: {
+      robotsTxtOne: get("/robots-one.txt", ({ ctx }) => {
```

Comment:
@luisherranz I run into this case when using Koa router middleware, the handler passed as a parameter doesn't have the types defined for `Server` or `AsyncServer`. I think we would need a different type for this case, but not sure how to proceed here.

I had a quick look at the `koa-route` types, but there isn't a type that allows passing a handler definition. Any ideas?

This might happen with other middleware, so I guess we need to create a generic one, instead of a solution for `koa-route`.

Reply:
@luisherranz I fixed this casting the types on the `koa-route` middleware handlers.

there was something else you mentioned that needed my attention, but cannot remember. Otherwise this implementation should be ready.